awk, cut, ed, sed: add and refresh pages (#7551)
tldr-pages/tldr: pages/osx/sed.md

# sed

> Edit text in a scriptable manner.
> See also: `awk`, `ed`.
> More information: <https://manned.org/man/freebsd-13.0/sed.1>.

> - Execute commands:

Reviewer: This is not correct. `expression` is the more accurate technical language here.
```suggestion
- Execute an expression:
```

`sed '{{s/apple/mango/g}}'`

- Execute a script:

`sed -f {{path/to/script.sed}}`

> - Execute commands with enabled [E]xtended regular expressions:

Reviewer: Same here.
```suggestion
- Execute an expression with enabled [E]xtended regular expressions:
```

`sed -E '{{s/(apple)/\U\1/g}}'`

> - Execute commands without automatic buffer printing:

Reviewer: Same here.
```suggestion
- Execute am expression without automatic buffer printing:
```

`sed -n '{{1p}}'`

> - Execute commands and replace file with it's output:

Reviewer: Same here.
```suggestion
- Execute an expression and replace the file with the output:
```

# sed

> Edit text in a scriptable manner.
> More information: <https://manned.org/sed.1>.

- Execute an expression:

> `sed "{{expression}}"`

Reviewer: I feel a number of the new examples on this page are too generic? I'm not convinced that the new layout of this page is more useful than the old one - though I do recognise the merits of showing the generic syntax first.

Author: I don't know how to escape placeholder character to write AWK code sample for:

```
`awk '{{{ print $1 }}}'`
```

It is useful because it shows difference between sed and awk.

Author: Fixed. ;)
